### DISCUSSION TITLE
Add PLUGINS.MD

### DISCUSSION THREAD

**Discussion by WASasquatch** [comfyanonymous/ComfyUI#441] [Ideas]
Add a `PLUGINS.MD` that users can PR, following the structure of the document. They could PR their GitHub to their Extension, Node, or "Other" plugins that extend the functionality of ComfyUI

**Comment by comfyanonymous** [comment]
Not sure if it belongs in the main repo. I can link it from the main repo and the examples page but it would probably best if it was somewhere else.

**Reply by WASasquatch** [reply]
What's the reasons? I feel it should be a MD file right on the repo to view as parsed markdown, and ship with the current versions for those keeping up to date. It doesn't make sense to have to follow another repo or place. Are you trying to commercialize and sell the product and trying to keep away from liabilities? Otherwise it seems best as part of the community github project structure. 

FYI it's weird enough going to another repo for examples on this repo. Very counter intuitive. Documentation should be here, wehter using the wiki module, or part of MD files to view, in this repo, regarding this repo. Ya know what I mean?

**Reply by comfyanonymous** [reply]
I like keeping things separate and organized. 

The main reason the examples are in another repo is because I don't think people who clone the main repo want to download 50MB+ (and increasing) of PNG files but it's also for organization.

**Reply by WASasquatch** [reply]
> I like keeping things separate and organized.
> 
> The main reason the examples are in another repo is because I don't think people who clone the main repo want to download 50MB+ (and increasing) of PNG files but it's also for organization.

Don't use PNG images, or use a host. That shouldn't really be a reason. ***(from experience, not all get the image thing as is and are confused by that, additionally, the images themselves should probably link the JSON version)***

And organized would imply with the project repo in question. 

It doesn't seem logical to the end user, and a weird preference. No offense. 

Also, 50mb off the size of the portable/dependencies is really moot. 

If *this were* a product, it wouldn't be legal to sell in many states because it's documentation/examples aren't shipped with it.

**Reply by WASasquatch** [reply]
Also, the wiki isn't downloaded with a repo, and would be a good place to move this, then you could just add an optional flag to the install process/portable that if the user so chooses, would clone the wiki with the project. Such as `--include-documentation` or w/e

![image](https://user-images.githubusercontent.com/1151589/230689292-03d5e8c4-eb63-44e8-af66-74de59a759de.png)

Wiki's have their own repo to clone, but are part of the project.

**Comment by WASasquatch** [comment]
@comfyanonymous https://github.com/WASasquatch/comfyui-plugins
